projects/admin/src/app/service/items.service.ts

```diff
@@ -41,6 +41,17 @@ export class ItemsService {
     private _apiService: ApiService
   ) { }
 
+  /**
+   * Get item by pid
+   * @param pid - String
+   * @returns Observable<any>
+   */
+  getEsItemByPid(pid: string): Observable<any> {
```

Comment:
I would rename this to `getByPidFromEs` or similar, `Item` is already in the service name.